src/boot/efifirmware.c

```diff
@@ -0,0 +1,89 @@
+/* SPDX-License-Identifier: LGPL-2.1-or-later */
+
+#include "efifirmware.h"
+#include "util.h"
+#include <endian.h>
+
+static bool efifw_validate_header(
+                const void *blob,
+                size_t uki_efifw_len,
```

Comment:
maybe rename this 2nd parameter to blob_len? because it is the length of the memory `blob` points to, no?